test/utils/Processing.test.ts

```diff
@@ -54,161 +50,17 @@ import {
   Configuration
 } from '../../src/fshtypes';
 import { EOL } from 'os';
+import { PREDEFINED_PACKAGE_NAME, PREDEFINED_PACKAGE_VERSION } from '../../src/ig';
+import { getTestFHIRDefinitions } from '../../test/testhelpers';
+import { InMemoryVirtualPackage, RegistryClient } from 'fhir-package-loader';
+import { logMessage } from '../../src/utils';
+import { mock, MockProxy } from 'jest-mock-extended';
 
-const NUM_R4_AUTO_DEPENDENCIES = 3;
+const NUM_R4_AUTO_DEPENDENCIES = 4;
 const NUM_R5_AUTO_DEPENDENCIES = 3;
 
-const TERM_R4_PKG_RESPONSE = {
```

Comment:
I am a big fan of how much simpler these test definitions get to be now.

Reply:
Me too!